css/_components/modal/_modal.css

```diff
@@ -0,0 +1,139 @@
+.modal__container {
+  align-items: center;
+  background-color: rgba(0, 0, 0, 0.2);
+  display: none;
+  height: 100vh;
+  justify-content: center;
+  left: 0;
+  position: fixed;
+  top: 0;
+  width: 100vw;
+}
+
+.modal {
+  background-color: white;
+  border-radius: var(--br-sm);
+  box-shadow: var(--bs-md);
+  flex-direction: column;
+  padding: var(--pd-sm);
+  width: min(85%, 500px);
+  min-width: 350px;
```

Comment:
Can clamp property be used here?

Reply:
Yes, it could be but I'm not very much comfortable with that as I haven't used it much.